helper/restore_helper.go

```diff
@@ -47,8 +48,33 @@ var (
 type RestoreReader struct {
 	bufReader  *bufio.Reader
 	seekReader io.ReadSeeker
+	pluginCmd  *pluginCmd
 	readerType ReaderType
-	errBuf     bytes.Buffer
+}
+
+// Log plugin's stderr as warnings.
+func (r *RestoreReader) logPlugin() {
+	if r.pluginCmd != nil {
+		errLog := strings.Trim(r.pluginCmd.stderrBuffer.String(), "\x00")
```

Comment:
It looks that stderrBuffer.String() doesn't flush out the content of buffer. At least this sample code:
```
	var stdError bytes.Buffer
	stdError.WriteString("Some_error")
	fmt.Println("ERROR:", stdError.String())
	fmt.Println("ERROR:", stdError.String())
```
results in:
```
ERROR: Some_error
ERROR: Some_error
```

So, once we got something into the stderr, we will print it every time. No?

Reply:
Thanks. We should consume the buffer contents on every log.